backends/dpdk/dpdkAsmOpt.h

```diff
@@ -135,6 +135,27 @@ class RemoveLabelAfterLabel : public Transform {
     }
 };
 
+
+// This pass Collects all metadata struct member used in program
+class CollectUsedMetadataField : public Inspector {
```

Comment:
Alternatively, if this pass is used only within dpdkAsmOpt.cpp, it does not need to have a declaration in the header, it can be local to the file (within an empty namespace to avoid name clashes).

Reply:
Now it's used in backend.cpp